frontend/components/color_encoding_to_sound/PaletteColor.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import SliderInstrument from "../instruments/SliderInstrument";
+
+class PaletteColor extends React.Component {
```

Comment:
For a component that has no state, like this one, consider using a [functional component](https://reactjs.org/docs/components-and-props.html) instead.

```
const PaletteColor = ({selected, color, id}) => {
    // does all of the same stuff as the render function
}
```